src/storage/compression/bitpacking_utils.cpp

```diff
@@ -0,0 +1,140 @@
+// packSingle/unpackSingle are adapted from
+// https://github.com/duckdb/duckdb/blob/main/src/storage/compression/bitpacking_hugeint.cpp
+
+#include "storage/compression/bitpacking_utils.h"
+
+#include "common/utils.h"
+
+namespace kuzu::storage {
+namespace {
```

Comment:
I made an anonymous namespace for the internal functions, I can remove if it's not part of our coding standards

Reply:
hmm I don't think we have that convention, but not against adding that either. @benjaminwinger what do you think?

Reply:
I'm fine with it. It's similar to making functions static, but somewhat more capable since you can also use it for classes/structs which are local to the translation unit.